src/backend/distributed/commands/function.c

```diff
@@ -1088,47 +1087,6 @@ EnsureSequentialModeForFunctionDDL(void)
 }
 
 
-/*
- * TriggerSyncMetadataToPrimaryNodes iterates over the active primary nodes,
- * and triggers the metadata syncs if the node has not the metadata. Later,
- * maintenance daemon will sync the metadata to nodes.
- */
-static void
-TriggerSyncMetadataToPrimaryNodes(void)
```

Comment:
In this PR I only moved the function to another file. But, i'm also considering to factor in `EnableMetadataSyncByDefault` GUC, in a separate PR. We should *probably* not sync metadata if ` EnableMetadataSyncByDefault = false`, but I need to think a little more

Reply:
I missed that we do not actually call this function in citus_disable_node.

Why did we move it? This logic seems very specific to the create_distributed_function implementation. (but probably obsolete)

Reply:
Hmm, probably tried for citus_disable_node and decided not to use, but forgot here. Let me move back